src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/generated/operators.ts

```diff
@@ -5474,8 +5122,6 @@ export const operatorFunctionDefinitions = [
   notInDefinition,
   notLikeDefinition,
   notRlikeDefinition,
-  isNullDefinition,
```

Comment:
If these are not exported here then where are we defining them?

Reply:
We will have to restore the manual definitions, but that doesn't have to do with the sync job